switch/mman.h

```diff
@@ -0,0 +1,125 @@
+#ifndef MMAN_H
```

Comment:
Move to 3rdParty folder

Reply:
fixed in https://github.com/diasurgical/devilutionX/pull/201/commits/2135690e21ec26830f9e2ee896c3eeb124366f8a